sympy/physics/control/lti.py

```diff
@@ -33,90 +32,171 @@ def _roots(poly, var):
+    num = Add(*[ T**(N-i) * c * (z-1)**i * (alpha * z + 1 - alpha)**(N-i) for c, i in zip(np[::-1], range(len(np))) ])
+    den = Add(*[ T**(N-i) * c * (z-1)**i * (alpha * z + 1 - alpha)**(N-i) for c, i in zip(dp[::-1], range(len(dp))) ])
 
-def backward_diff(tf, sample_per):
-        """
-        Returns falling coefficients of H(z) from numerator and denominator.
-        Where H(z) is the corresponding discretized transfer function,
-        discretized with the backward difference transform method.
-        H(z) is obtained from the continuous transfer function H(s)
-        by substituting s(z) =  (z-1)/(T*z) into H(s), where T is the
-        sample period.
-        Coefficients are falling, i.e. H(z) = (az+b)/(cz+d) is returned
-        as [a, b], [c, d].
+    num_coefs = num.as_poly(z).all_coeffs()
+    den_coefs = den.as_poly(z).all_coeffs()
 
-        Examples
-        ========
+    para = den_coefs[0]
+    num_coefs = [coef/para for coef in num_coefs]
+    den_coefs = [coef/para for coef in den_coefs]
 
-        >>> from sympy.physics.control.lti import TransferFunction, backward_diff
-        >>> from sympy.abc import s, L, R, T
-        >>> tf = TransferFunction(1, s*L + R, s)
-        >>> numZ, denZ = backward_diff(tf, T)
-        >>> numZ
-        [T, 0]
-        >>> denZ
-        [L + R*T, -L]
-        """
+    return num_coefs, den_coefs
 
-        T = sample_per  # and sample period T
-        s = tf.var
-        z =  s         # dummy discrete variable z
+def bilinear(tf, sample_per):
+    r"""
+    Returns falling coefficients of H(z) from numerator and denominator.
 
-        np = tf.num.as_poly(s).all_coeffs()
-        dp = tf.den.as_poly(s).all_coeffs()
+    Explanation
+    ===========
 
-        # The next line results from multiplying H(z) with z^N/z^N
+    Where H(z) is the corresponding discretized transfer function,
+    discretized with the bilinear transform method.
+    H(z) is obtained from the continuous transfer function H(s)
+    by substituting $s(z) = \frac{2}{T}\frac{z-1}{z+1}$ into H(s), where T is the
+    sample period.
+    Coefficients are falling, i.e. $H(z) = \frac{az+b}{cz+d}$ is returned
+    as [a, b], [c, d].
 
-        N = max(len(np), len(dp)) - 1
-        num = Add(*[ T**(N-i)*c*(z-1)**i*(z)**(N-i) for c, i in zip(np[::-1], range(len(np))) ])
-        den = Add(*[ T**(N-i)*c*(z-1)**i*(z)**(N-i) for c, i in zip(dp[::-1], range(len(dp))) ])
+    Examples
+    ========
 
-        num_coefs = num.as_poly(z).all_coeffs()
-        den_coefs = den.as_poly(z).all_coeffs()
+    >>> from sympy.physics.control.lti import TransferFunction, bilinear
+    >>> from sympy.abc import s, L, R, T
 
-        return num_coefs, den_coefs
+    >>> tf = TransferFunction(1, s*L + R, s)
+    >>> numZ, denZ = bilinear(tf, T)
+    >>> numZ
+    [0.5*T/(1.0*L + 0.5*R*T), 0.5*T/(1.0*L + 0.5*R*T)]
+    >>> denZ
+    [1, (-1.0*L + 0.5*R*T)/(1.0*L + 0.5*R*T)]
+    """
+    return gbt(tf, sample_per, 0.5)
```

Comment:
It is better to use the symbolic rationals than floats
```suggestion
    return gbt(tf, sample_per, S.Half)
```